### DISCUSSION TITLE
CCU3 als Docker-Host mit Raspberrymatic

### DISCUSSION THREAD

**Discussion by albatrosify** [OpenCCU/OpenCCU#2020] [🇩🇪 Fragen&Antworten]
Hallo Zusammen,

ich würd gerne meine CCU3 als Docker-Host "missbrauchen" und Pi-Hole + Rapsberrymatic drauf laufen lassen.
Die original-CCU hat ja das original-Funkmodul über den GPIO. Ist dieses mit Raspberrymatic in einem Docker-Container kompatibel?

Ich hatte auf einer 2. SD Karte das Raspberry OS Lite installiert + Docker und darin Raspberry-Matic aber leider wurde das original Funkmodul nicht gefunden.

Vielleicht kann jemand kurz helfen ?

Merci <3

**Comment by Baxxy13** [comment]
Das [Wiki zur Docker-Installation](https://github.com/jens-maus/RaspberryMatic/wiki/Installation-Docker-OCI) hast du gelesen und penibel befolgt?

Müsste eigentlich gehen.

Ich bin kein Freund von "Docker-RaspberryMatic", hatte das aber mal mit einer USB-Funkmodul-Lösung (in einer Debian-VM auf Pimox) probiert und das lief.

**Reply by albatrosify** [reply]
Vielen Dank für die schnelle Antwort.
Ich habe es nach bestem Gewissen befolgt. Allerdings finde ich in der Beschreibung nicht das passende Funkmodul. Welche Teilenummer hat denn das Original CCU3 Funkmodul? Es wird im Wiki nur von irgendwelchen anderen Teilen gesprochen.

👋

**Reply by jens-maus** [reply]
In der CCU3 steckt natürlich das RPI-RF-MOD.
